### PR TITLE
Restyle Transaction and Block detail pages

### PR DESCRIPTION
**Pull request by mr-pavlenko**
Restyle two pages according to the new design:
- Transaction Detail
- Block Detail

Screenshot 2022-08-27 at 11 28 15
Screenshot 2022-08-27 at 11 28 38

### CODE REVIEW

File: src/views/TransactionsItem.vue [hunk 1]
```diff
@@ -18,144 +22,86 @@
...
+              <div class="app-table__cell app-table__cell-label">
+                <InfoIcon :message="TOOLTIP_INFO.status" />
+                <span>Status</span>
               </div>
-              <span class="transactions-item__table-row-title">Status</span>
-              <div class="transactions-item__table-row-value">
+              <div class="app-table__cell">
                 <span
-                  class="transactions-item__table-row-status"
+                  class="tx-item__status"
```
Comment (nikita1tup) [inline src/views/TransactionsItem.vue:47]: transactions-item__status

File: src/views/TransactionsItem.vue [hunk 2]
```diff
@@ -18,144 +22,86 @@
...
-                {{ tx.gasUsed }} / {{ tx.gasWanted }}
-              </span>
             </div>
-            <div class="transactions-item__table-row">
-              <div class="transactions-item__table-row-info">
-                <img src="~@/assets/icons/info.svg" alt="info" />
-                <span class="transactions-item__table-row-tooltip">
-                  {{ TOOLTIP_INFO.fee }}
-                </span>
+            <div class="app-table__row tx-item__line">
```
Comment (nikita1tup) [inline src/views/TransactionsItem.vue:76]: transactions-item

**Review by nikita1tup — CHANGES_REQUESTED** [review]

File: src/views/TransactionsItem.vue (hunk 1 above)
Comment (mr-pavlenko) [reply src/views/TransactionsItem.vue]: done

File: src/views/TransactionsItem.vue (hunk 2 above)
Comment (mr-pavlenko) [reply src/views/TransactionsItem.vue]: done

**Pull request merged**